src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/MapPost_WithArrayQueryString_AndBody_ShouldUseBody.generated.txt

```diff
@@ -97,34 +97,21 @@ namespace Microsoft.AspNetCore.Http.Generated
                     Debug.Assert(options != null, "RequestDelegateFactoryOptions not found.");
                     Debug.Assert(options.EndpointBuilder != null, "EndpointBuilder not found.");
                     options.EndpointBuilder.Metadata.Add(new SourceKey(@"TestMapActions.cs", 25));
-                    var serviceProvider = options.ServiceProvider ?? options.EndpointBuilder.ApplicationServices;
-                    var serviceProviderIsService = serviceProvider.GetRequiredService<IServiceProviderIsService>();
-                    var jsonBodyOrServiceTypeTuples = new (bool, Type)[] {
-                        (false, typeof(global::System.String[])),
-                    };
-                    foreach (var (isOptional, type) in jsonBodyOrServiceTypeTuples)
```

Comment:
Will this still emit the correct metadata for the request body?

Reply:
A good catch. I'll need to add a test to cover this.